src/main/java/uk/gov/ons/ctp/response/collection/exercise/CollectionExerciseApplication.java

```diff
@@ -44,6 +51,9 @@
 @ComponentScan(basePackages = {"uk.gov.ons.ctp.response"})
 @EnableJpaRepositories(basePackages = {"uk.gov.ons.ctp.response"})
 @EntityScan("uk.gov.ons.ctp.response")
+@EnableScheduling
+@EnableCaching
+@EnableAsync
```

Comment:
Why enable async? We're not using it anywhere so far as I can see, and it can have quite unpredictable results sometimes.

Reply:
Will remove it 👍